database/buildSchema/16_template_element.sql

```diff
@@ -52,6 +57,8 @@ CREATE TABLE public.template_element (
     validation_message varchar,
     help_text varchar,
     parameters jsonb,
+    is_reviewable public.is_reviewable_status DEFAULT NULL,
+    -- review_required boolean NOT NULL DEFAULT TRUE,
```

Comment:
I really wanted to add this field to make certain elements *optionally* reviewable, but the impact on the front-end logic was too much to risk right now, as I'm pretty sure it'd introduce unforeseen regressions, so have just left it as a "TO-DO" for now.